src/store/holochain.js

```diff
@@ -285,14 +301,16 @@ export default {
       state.reconnectingIn = -1
       state.firstConnect = false
     },
-    setHoloClientAndDnaAlias (state, payload) {
-      const { holoClient, dnaAlias } = payload
-      console.log('HOLO DNA ALIAS : ', dnaAlias)
-      state.dnaAlias = dnaAlias
-      state.holoClient = holoClient
+    setHoloClient (state, payload) {
+      console.log('holo client: ', payload)
+      state.holoClient = payload
       state.reconnectingIn = -1
       state.firstConnect = false
     },
+    setDnaAlias (state, payload) {
+      console.log('holo dna alias : ', payload)
```

Comment:
remove log

Reply:
Same as above.  This log was previously there.  Do we no longer want to show this?

Reply:
hmm. Yeah, maybe we want to keep this around. But maybe we switch it to use the `log` function rather than `console.log`

```suggestion
      log('holo dna alias : ', payload)
```

That's a way I've been distinguishing between logs we want to keep around vs temporary debugging logs